src/flyff/flyffentity.js

```diff
@@ -1150,6 +1152,14 @@ export default class Entity {
             }
         }
 
+        // Party buffs
+        // Not sure if the calculation is correct.
+        for (const [, id] of Object.entries(this.activePartyBuffs)) {
+            if (stat == "partycriticalchance" && id == 5942) {
+                total += Utils.clamp(this.activePartyMembers, 1, 8) * 0.5
+            }
+        }
+
```

Comment:
I would just add this directly to `getCriticalChance` instead of making a fake stat here. The calculation is correct though 👍